src/common/vfs.cpp

```diff
@@ -72,7 +72,7 @@ bool OCC::isVfsPluginAvailable(Vfs::Mode mode)
     auto name = modeToPluginName(mode);
     if (name.isEmpty())
         return false;
-    return PluginLoader().load("vfs", name);
+    return QPluginLoader(pluginFileName("vfs", name)).load();
```

Comment:
Is there not an option to do that without actually loading the plugin?  Perhaps querying the metadata.
Also, don't we need a mirall version check?

Reply:
Yeah, I think it makes a lot of sense to ensure the plugin and client version match exactly, and that can be done with metadata without loading the plugin. I'll check it out.